components/forms/AddForm.js

```diff
@@ -0,0 +1,199 @@
+        throw new Error(`Failed to add plant: ${response.statusText}`);
+      }
+
+      const newPlant = await response.json();
+      console.log("Plant added successfully:", newPlant);
+
+      router.push(`/`);
+    } catch (error) {
+      console.error("Error adding plant:", error);
+      alert("Failed to add plant. Please try again.");
+    }
+  }
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData.entries());
+    const fertiliserSeasons = formData.getAll("fertiliserSeason");
+    const dataWithSeasons = { ...data, fertiliserSeason: fertiliserSeasons };
+    console.log(dataWithSeasons);
+
+    // ONLY IMAGES FROM UNSPLASH FOR NOW
+    if (!data.imageUrl.startsWith("https://images.unsplash.com")) {
+      alert("Image URL must start with https://images.unsplash.com/");
+      return;
+    }
+    if (fertiliserSeasons.length === 0) {
+      alert("Please choose at least one Fertiliser Season!")
+      return;
+    }
+    addPlant(dataWithSeasons);
+  }
+
+  return (
+    <Form onSubmit={handleSubmit}>
+      <h2>Add a new plant</h2>
+
+      <Label>
+        Name
+        <Input name="name" type="text" required />
+      </Label>
+
+      <Label>
+        Botanical Name
+        <Input name="botanicalName" type="text" required />
+      </Label>
+
+      <Label>
+        Image URL
+        <Input name="imageUrl" type="text" required />
+      </Label>
+
+      <Fieldset>
+        <legend>Water Need</legend>
+        <Slider name="waterNeed" type="range" min="1" max="3" step="1" />
+        <Scale>
+          <span>💧</span>
+          <span>💧💧</span>
+          <span>💧💧💧</span>
+        </Scale>
+      </Fieldset>
+
+      <Fieldset>
+        <legend>Light Need</legend>
+        <Slider name="lightNeed" type="range" min="1" max="3" step="1" />
+        <Scale>
+          <span>☁️</span>
+          <span>🌥️</span>
+          <span>☀️</span>
+        </Scale>
+      </Fieldset>
+
+      <Fieldset>
+        <legend>Fertiliser Season</legend>
+        <CheckboxLabel>
+          <input type="checkbox" name="fertiliserSeason" value="spring" />
+          🌸
+        </CheckboxLabel>
+        <CheckboxLabel>
+          <input type="checkbox" name="fertiliserSeason" value="summer" />
+          ☀️
+        </CheckboxLabel>
+        <CheckboxLabel>
+          <input type="checkbox" name="fertiliserSeason" value="autumn" />
+          🍁
+        </CheckboxLabel>
+        <CheckboxLabel>
+          <input type="checkbox" name="fertiliserSeason" value="winter" />
+          ❄️
+        </CheckboxLabel>
+      </Fieldset>
+
+      <Label>
+        Description
+        <Textarea name="description" rows="4" required />
+      </Label>
+
+      <Button type="submit">Save Plant</Button>
+    </Form>
+  );
+}
+
+/* styled-components */
+const Form = styled.form`
+  padding: 1.5rem;
+  h2 {
+    margin-bottom: 1rem;
+    text-align: center;
+  }
```

Comment:
please avoid nested rulesets whenever possible. This has a different specificity than other rulesets which can cause problems down the line.